apps/backend/src/db.ts

```diff
@@ -0,0 +1,226 @@
+import { Status, success, error, UserBase, WorkoutBase } from '@dundring/types';
+import {
+  FitnessData,
+  PrismaClient,
+  SteadyWorkoutPart,
+  User,
+  Workout,
+} from '@prisma/client';
+export const prisma = new PrismaClient();
+
+export const getUserByUsername = async (
+  username: string
+): Promise<
+  Status<
```

Comment:
I tried creating a type ala
```typescript
type AsyncStatus<T, E> = Promise<Status<T, E>>
```
but that isn't allowed 😞 

```
Error: Type 'AsyncStatus' is not a valid async function return type in ES5/ES3 because it does not refer to a Promise-compatible constructor
```